pandas/core/apply.py

```diff
@@ -404,17 +427,18 @@ def agg_dict_like(self) -> DataFrame | Series:
         )
 
         # Numba Groupby engine/engine-kwargs passthrough
-        kwargs = {}
         if is_groupby:
             engine = self.kwargs.get("engine", None)
             engine_kwargs = self.kwargs.get("engine_kwargs", None)
-            kwargs = {"engine": engine, "engine_kwargs": engine_kwargs}
+            kwds.update({"engine": engine, "engine_kwargs": engine_kwargs})
```

Comment:
NBD, but I wonder why the change from kwargs to kwds? In pandas.core we overwhelmingly use kwargs instead of kwds.

Reply:
kwargs would make a line further down excedd 88 lines and be reformatted to fill 3 lines. So a stylistic preference, but not a strong opinion,

Reply:
I think consistency in variable names is more important here.